core/test/gather/gatherers/full-page-screenshot-test.js

```diff
@@ -202,4 +208,338 @@ describe('FullPageScreenshot gatherer', () => {
       }
     );
   });
```

Comment:
These tests are good, but IMO should completely replace the old test cases. I understand not doing that to make the diff cleaner in this PR tho.

Reply:
The existing ones are only as good as our ability to simulate how the browser handles DPR, page scale, and CDP emulation together. IMO we are never going to get that right so we should just rely on these new e2e tests (and smoke tests) to get this right.

Reply:
Bump, could be a follow-up but I still think the old test cases should be converted to smoke tests or this kind of test